class02/exercise_6.py

```diff
@@ -23,6 +23,9 @@ def calculation(a, b):
 	'''
 
 	# code up your solution here
+	summation = a + b
+	diff = a - b
+	return[summation, diff]
```

Comment:
this is returning a list, think how we can return two numbers

Reply:
Fixed. Now returns a tuple.
Thanks for reviewing the homework.